developer_tests/normal_dist/work/quickbuild.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
```

Comment:
Unrelated to the content of this PR, but I don't think we want to keep adding new files to the repo with this header since this webpage no longer exists.

See [this PR](https://github.com/NCAR/DART/pull/617)

Reply:
We do need a copyright statement, and we don't have an alternative at the movement.  I'd rather we had the same header in each file and we replace them all at once. If you look at issue #616 there are different headers that people are putting in the code for ?reasons?.
Personally, I think the short version is the way to go, 
```
Copyright [yyyy] [name of copyright owner]
SPDX-License-Identifier: Apache-2.0 
```
Example of musica  using the short version https://github.com/NCAR/musica/blob/d8c5aae03d5e486883b1fcc2e6c6e27ce126111d/src/micm/micm.cpp#L1-L2